client/src/Pages/User/Orders/index.js

```diff
@@ -2,120 +2,271 @@ import React, { useState } from "react";
+        </span>
+      ))}
+      <hr class="solid"></hr>
+    </div>
+  );
+};
+
+const TotalAndTax = ({ items }) => {
+  var tax = 0.1;
+  var total = 0;
+  var list = items.map((item) => {
+    if (parseInt(item.product.price) != undefined) {
+      return parseFloat(item.product.price);
+    } else {
+      return 0.0;
+    }
+  });
+
+  for (const item_order of list) {
+    total += parseFloat(item_order);
+  }
+
+  return (
+    <div>
+      <p>
+        Tax: <strong>${roundMoney(total * tax)}</strong>
+      </p>
+      <p>
+        Total: <strong>${roundMoney(total + total * tax)}</strong>
+      </p>
+    </div>
+  );
 };
 
 const OrderDetail = ({ order }) => {
-    const { _id, items, vendor, user } = order;
-    const [modalOpen, setModalOpen] = useState(false);
-
-    const openModal = () => setModalOpen(true);
-    const closeModal = () => setModalOpen(false);
-
-	const [cancelOrder] = useMutation(CANCEL_ORDER);
-
-	const handleCancelOrder = () => {
-		cancelOrder({ variables: { _id: _id } });
-	};
-
-	return (
-		<div className="ordercard">
-			<h2>{order.fulfillment}</h2>
-			<p>{vendor.name}</p>
-            <button onClick={openModal}>Show Order Items</button>
-			{order.fulfillment == "Placed" ? (
-				<button onClick={handleCancelOrder}>Cancel Order</button>
-			) : null}
-            <Modal
-            className="modal"
-            isOpen={modalOpen}
-            onRequestClose={closeModal}
-            >
-                <OrderItemList items={items} />
-            </Modal>
-		</div>
-	);
+  const { _id, items, vendor, user, createdAt } = order;
+  const [modalOpen, setModalOpen] = useState(false);
+  const [detailOpen, setDetailOpen] = useState(false);
+
+  const openModal = () => setModalOpen(true);
+  const closeModal = () => setModalOpen(false);
+
+  const [cancelOrder] = useMutation(CANCEL_ORDER);
+
+  const handleCancelOrder = () => {
+    cancelOrder({ variables: { _id: _id } });
+  };
+
+  function detailsClick() {
+    if (detailOpen) {
+      setDetailOpen(false);
+    } else {
+      setDetailOpen(true);
+    }
+  }
+
+  function DateFormatter({ date }) {
+    var formattedDate = date.split("-");
+    var months = [
+      "January",
+      "February",
+      "March",
+      "April",
+      "May",
+      "June",
+      "July",
+      "August",
+      "September",
+      "October",
+      "November",
+      "December",
+    ];
+    return (
+      <p>
+        {months[parseInt(formattedDate[1]) - 1]}{" "}
+        {formattedDate[2].substring(0, 2)}, {formattedDate[0]}
+      </p>
+    );
+  }
+
+  function vendorLogo(name) {
+    if (name == "East West Tea") {
+      console.log(true);
+      return "//static1.squarespace.com/static/58559451725e25a3d8206027/t/58559539f5e2315e3ef1127c/1593500228704/?format=1500w";
+    }
+    //CoffeeHouse
+    //Grillosophy
+  }
+
+  return (
+    <div className="ordercard">
+      <div className="orderText">
+        <img src={vendorLogo(vendor.name)} className="vendorLogo" />
```

Comment:
on iPhone X dimensions the image appears above the vendor name instead of next to it, as per the design.